custom/icds_reports/tasks.py

```diff
@@ -327,6 +332,12 @@ def _aggregate_thr_forms(state_id, day):
     AggregateChildHealthTHRForms.aggregate(state_id, day)
 
 
+@track_time
+def _aggregate_inactive_aww(day):
+    one_day_before = day - relativedelta(days=1)
```

Comment:
Instead of having this always be one day, can we save the time that the aggregation begins and then use that to pass to the aggregate function? That way we don't miss any forms in case the task is delayed for ay reason